setup.cfg

```diff
@@ -0,0 +1,2 @@
+[tool:pytest]
+addopts =-x -v -rw --durations=10 --cov=spyder --cov-report=term-missing
```

Comment:
>  (if possible)

you could either define an environment variable [`PYTEST_ADDOPTS`](https://docs.pytest.org/en/latest/customize.html#adding-default-options) in the CI scripts, or rename the config file to, e.g., `pytest-ci.cfg` and use the `-c file` command line flag to discover *that* config file.